packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -14,10 +14,13 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
+  url_launcher_platform_interface: ^1.0.0
```

Comment:
If this lands after #2230 lets make this ^1.01

Reply:
Done.